appinventor/lib/blockly/src/core/xml.js

```diff
@@ -347,27 +350,6 @@ Blockly.Xml.domToBlockInner = function(workspace, xmlBlock, opt_reuseBlock) {
     block.initSvg();
   }
 
-  var inline = xmlBlock.getAttribute('inline');
```

Comment:
@halatmit I just checked the latest blockly, and all of this code is refactored. Not sure if that should change our action here. We *have* bad blocks, blockly doesn't. So we will have to re-integrate them into the latest blockly when we merge it back. The Good News is it shouldn't be hard.